### ISSUE TITLE
Fixed bookmark bar

### ISSUE THREAD

**Issue by gmuder** [minbrowser/min#1159]
Is it possible to leave the bookmark bar fixed somewhere in the browser screen?

It is not practical to click on the 3 dot menu and select bookmark. I use it a lot.

**Comment by Syndamia** [comment]
For now, no, the [wiki](https://github.com/minbrowser/min/wiki#bookmarks) doesn't say anything about it and I can't find an option in the UI.

**Comment by PalmerAL** [comment]
Yeah, there isn't an option for a fixed bar, and I'd prefer not to add one. In theory, the list of top sites that shows up when you open the searchbar should help replace it, but I'm not sure how effective it actually is; we might want to consider making other changes (showing more results, other types of results, ...) to it.

**Comment by Andy2244** [comment]
I second this request, i need a Edge replacement that start's quickly and is lean and fast for my lower end devices. Yet i need the ability for a fixed bookmark bar, similar to Edge/Chrome, where i can quickly access all my web-based media-pc services. Relying on the auto history or always needing to Menu->Bookmark->search&click is just to slow and cumber-son.

So please add a fixed bookmarks bar as option.
